src/routes/log.js

```diff
@@ -11,6 +11,13 @@ function log(req, res) {
   });
 }
 
-router.get('/', log);
+async function write(req, res) {
```

Comment:
What's the use case for writing logs to device? Is this just for better end-to-end tests for reading logs?